subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -229,3 +229,31 @@ Run with `--watch-fs` on the command line::
 This enables watching the file-system for this build only.
 Put `org.gradle.unsafe.watch-fs=true` in your `gradle.properties`::
 This enables watching the file-system for all builds, unless explicitly disabled with `--no-watch-fs`.
+
+[[sec:daemon_watch_fs_troubleshooting]]
+=== Troubleshooting file-system watching
+
+Limitations::
+- If your build doesn’t declare inputs and outputs properly, there is a chance Gradle does not detect changes to undeclared stuff.
+- If you have symlinks in your build, you won’t get the performance benefits for those locations (we plan to change this in the future).
+- Default excludes (ignoring files and directories like “CVS/”, “.DS_Store” etc.) cannot be configured when VFS retention is enabled.
+- When multiple daemons are running, the idle ones can pick up the changes produced by the others and create large log files with lots of debug info about the changes.
+- On Windows, we don’t support SUBST and network drives (they might work, but we don’t test them yet).
+
+Gradle does not pick up some of my changes.::
+_Please https://gradle-community.slack.com/app_redirect?channel=file-system-watching[let us know] if that happens to you._ If your build declares its inputs and outputs correctly, this should not happen. If it does, it’s either a bug we need to fix, or your build is lacking the declaration of some inputs or outputs.
+
+Why am I always getting “Received 8 file system events since last build” even though I only changed one file?::
+These are harmless notifications about changes to Gradle's own caches that happen after file watching has started.
```

Comment:
Do we want to mention that this is going away soon?

Reply:
I wouldn't. If they are gone, let's remove them from the user manual.

Reply:
I wouldn't. If they are gone, let's remove them from the user manual.